plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -838,7 +838,12 @@ public synchronized void dropRole(String role)
     @Override
```

Comment:
Let's mention in the commit message that it was actually a bug because `listRoleGrants` was already returning admin role.